.gitignore

```diff
@@ -0,0 +1,2 @@
+# Ignore common IDE files
+/.idea
```

Comment:
We recommend putting IDE files in your global git ignore (https://help.github.com/articles/ignoring-files/)